doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -518,7 +518,7 @@ Trusted Firmware-M (TF-M) samples
 Thread samples
 --------------
 
-|no_changes_yet_note|
+* Enabled the :ref:`ug_thread_build_report` generation in the :ref:`ot_cli_sample` and :ref:`ot_coprocessor_sample`.
```

Comment:
```suggestion
* Enabled the :ref:`ug_thread_build_report` generation in the :ref:`ot_cli_sample` and :ref:`ot_coprocessor_sample` samples.
```
Not a blocker, but if you need to make any other changes, you could add this as well.